jerry-core/jcontext/jcontext.h

```diff
@@ -116,6 +117,80 @@ typedef struct
 } jerry_context_t;
 
 /**
+ * Description of jerry instance, which contains the meta data and buffer.
+ */
+struct jerry_instance_t
+{
+  struct
+  {
+    uint32_t heap_size; /**< size of the heap */
+    uint8_t *context_p; /**< point to the entrance of the context in buffer */
+    uint8_t *heap_p; /**< point to the entrance of the heap in buffer */
+    uint8_t *lcache_p; /**< point to the entrance of the lcache in buffer */
```

Comment:
CONFIG_ECMA_LCACHE_DISABLE / JERRY_SYSTEM_ALLOCATOR should be checked here as well.